README.md

```diff
@@ -97,9 +97,20 @@ If an individual source table has a different name than the package expects, add
 ```yml
 vars:
   pendo_source:
-    pendo_<default_source_table_name>_identifier: your_table_name 
+    pendo_<default_source_table_name>_identifier: "your_table_name" 
 ```
+🚨 Snowflake Users
+You may need to provide the case-sensitive spelling of your source tables that are also Snowflake reserved words.
 
+In this package, this would apply to the `GROUP` source. If you are receiving errors for this source, include the following in your `dbt_project.yml` file:
+
+```yml
+vars:
+  pendo_group_identifier: '"GrouP"' # as an example, must include this quoting pattern and your exact casing
```

Comment:
This read strange. It looks like we are recommending the customer to supply the variable with `GrouP`. Would we be able to just do `Group`. It is the same casing example, and may resemble a customers environment more accurately.

Reply:
Updated!